decidim-elections/app/commands/decidim/votings/census/admin/create_dataset.rb

```diff
@@ -31,7 +31,7 @@ def call
             end
 
             if dataset
-              CSV.foreach(form.file.tempfile.path, col_sep: ";", headers: true) do |row|
+              CSV.foreach(form.file.tempfile.path, col_sep: ";", headers: true, converters: ->(f) { f ? f.strip : nil }) do |row|
```

Comment:
💬  Can this code be written like this? or I'm missing something?
```suggestion
              CSV.foreach(form.file.tempfile.path, col_sep: ";", headers: true, converters: ->(f) { f&.strip }) do |row|
```

Reply:
Nope, I don't think you are missing anything. 😄